src/integration-test/java/org/openlmis/requisition/repository/RequisitionTemplateRepositoryIntegrationTest.java

```diff
@@ -391,4 +398,41 @@ private AvailableRequisitionColumn getColumnWithOption() {
   private AvailableRequisitionColumnOption getOption(String uuid) {
     return availableRequisitionColumnOptionRepository.findById(UUID.fromString(uuid)).orElse(null);
   }
+
+  @Test
+  public void shouldAllowDuplicationForRequisitionReportOnlyTemplateNames() {
+
```

Comment:
Empty lines at the beginning and at the end of the functions are inconsistent with the rest of the code. Please delete them in all added functions

Reply:
This has been resolved, please see the updated one.